assets/src/components/layout/Sidebar.tsx

```diff
@@ -77,51 +80,59 @@ function getMenuItems({
   return [
     {
       text: 'Home',
+      expandedLabel: 'Home',
       icon: <HomeIcon />,
       path: '/home',
     },
     {
       text: 'Apps',
+      expandedLabel: 'Apps',
       icon: <AppsIcon />,
       path: '/',
       plural: true,
       pathRegexp: /^\/(apps)/,
     },
     {
       text: 'Continuous deployment',
+      expandedLabel: 'CD',
```

Comment:
I know it's a little bit different but to keep things simpler I'd use `text` as `expandedLabel` and figure out which variant to use.